versioned_docs/version-3.2.x/getting-started-docker.md

```diff
@@ -56,6 +56,22 @@ bin/pulsar standalone
 </Tabs>
 ````
 
+:::tip
+
+You may encounter issues with the default RocksDB metadata store after restarting Pulsar standalone.
+
+We recommend you consider using the following environment variable to use ZooKeeper as the metadata store:
```

Comment:
I don't think it's a good idea to recommend not to use a default setting. If we have a bug in default configuration, we must fix it. If it cannot be fixed, we will have to change the defaults.

Reply:
I would be the best solution. @lhotari 's concern was that:

> it would have to be documented since many might be using old versions and wondering about the resolution.
> The problem in changing the default is that it would break existing "deployments" with data

Reply:
> I don't think it's a good idea to recommend not to use a default setting. If we have a bug in default configuration, we must fix it. If it cannot be fixed, we will have to change the defaults.

@merlimat I agree. Since we cannot change already released versions and users are hitting problems, I think it's worth documenting the workaround until there's a better solution. It seems that there isn't a nice way to solve this problem.

Reply:
I'll merge this PR with the current content. We can come back to this later once there's a better resolution.